### PR TITLE
Minor ReadMe improvement testing (#157), contributing, contact sections.

### PR DESCRIPTION
**Pull request by bvberkum**
This is just a minimal change to README to reflect current status.

----

- [x] I have reviewed the [Contributor Guidelines][contributor].
- [x] I have reviewed the [Code of Conduct][coc] and agree to abide by it

[contributor]: https://github.com/bats-core/bats-core/blob/master/docs/CONTRIBUTING.md
[coc]:         https://github.com/bats-core/bats-core/blob/master/docs/CODE_OF_CONDUCT.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -443,12 +453,20 @@ To learn how to set up your editor for Bats syntax highlighting, see [Syntax
...
 
+## Contributing
+
+For now see the ``docs`` folder for project guides, work with us on the wiki
+or look at the other communication channels.
+
+## Contact
+
+- We are `#bats` on freenode;
+- Or leave a message on [gitter].
```
Comment (GhostofGoes) [inline README.md]: You're missing the Gitter URL

Comment (GhostofGoes) [reply README.md]: Also, I can't find the `#bats` channel on freenode. Does it exist yet?

Comment (bvberkum) [reply README.md]: This is from original docs, I only moved it. Perhaps better point to (and update) wiki contact page.

Comment (bvberkum) [reply README.md]: Also the gitter URL works, the def is (was already) at the bottom.
See live version https://github.com/bvberkum/bats-core/tree/forks/bvb-docs

Comment (GhostofGoes) [reply README.md]: Ah, didn't realize that. No, keeping it in the README is probably the best IMO.

**Comment by sublimino** [comment]
LGTM, thanks @bvberkum :+1:

**Pull request merged**